Emby.AutoOrganize/Data/SqliteExtensions.cs

```diff
@@ -1,15 +1,55 @@
 using System;
 using System.Collections.Generic;
 using System.Globalization;
-using MediaBrowser.Model.IO;
+using System.IO;
 using MediaBrowser.Model.Serialization;
 using SQLitePCL.pretty;
-using System.IO;
 
 namespace Emby.AutoOrganize.Data
 {
     public static class SqliteExtensions
```

Comment:
The same files exists in the server, we should reduce all this duplicate code

Reply:
Created https://github.com/jellyfin/jellyfin-plugin-autoorganize/issues/32 to tackle this